cli/azd/cmd/down.go

```diff
@@ -28,25 +28,26 @@ func newDownCmd() *cobra.Command {
 }
 
 type downAction struct {
-	runner      middleware.MiddlewareContext
-	infraDelete infraDeleteAction
+	flags                  *downFlags
+	runner                 middleware.MiddlewareContext
+	infraDeleteInitializer actions.ActionInitializer[*infraDeleteAction]
```

Comment:
Do we need to do this for `down`? It's an alias so I don't think we need lazy -- but I could be wrong here.

Reply:
You are likely correct, I was just making it consistent for any actions that inject child actions.

Reply:
I was able to fully deprecate the duplicated alias commands for both `provision` and `down`.